crates/rari-doc/src/templ/templs/csssyntax.rs

```diff
@@ -58,6 +58,7 @@ pub fn csssyntax(name: Option<String>) -> Result<String, DocError> {
 
     Ok(render_formal_syntax(
         SyntaxInput::Css(typ),
+        env.browser_compat.first().map(|s| s.as_str()),
```

Comment:
Do you know if there are any pages using the formal syntax macro with multiple browser-compat frontmatter values?